docs/integrations/databases/opentelemetry/cassandra-opentelemetry.md

```diff
@@ -0,0 +1,171 @@
+- **`sumo.datasource`** - Has fixed value of **cassandra**.
+
+## Prerequisite
+
+
+Cassandra has three main logs: system.log, debug.log, and gc.log which hold general logging messages, debugging logging messages, and java garbage collection logs respectively.
+
+These logs by default live in ${CASSANDRA_HOME}/logs, but most Linux distributions relocate logs to /var/log/cassandra. Operators can tune this location as well as what levels are logged using the provided logback.xml file. For more details on Cassandra logs, see[ this](https://cassandra.apache.org/doc/latest/troubleshooting/reading_logs.html) link.
+
+## Configure Cassandra Logs Collection
+
+### Step1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the **Use an existing Collector** option.
+:::
+
+To create a new Collector:
+
+1. Select the **Add a new Collector** option.
+2. Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
+
+
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Cassandra-OpenTelemetry/Collector.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+In this step we will be configuring the yaml required for Cassandra Collection.
+Path of the log file configured to capture Cassandra logs needs to be given here.
+
+In this step we will be configuring the yaml required for Cassandra Collection.
+Below are the input required :
+-   The path to system.log is required here. This file is typically located in /var/log/cassandra. If you're using a customized path, check the respective conf file for this information.
+
+You can add any custom fields which you want to be tagged with the data ingested in sumo.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Cassandra-OpenTelemetry/YAML.png')} alt="YAML" />
+
+### Step 3: Sending logs to Sumo
+
+Once you have the yaml file downloaded in step 2, please follow the below steps based on your platform
+
+Linux:
+
+1.  Copy the yaml to `/etc/otelcol-sumo/conf.d/` folder for the Cassandra instance which needs to be monitored.
+2.  restart the collector using
+```sh
+ sudo systemctl restart otelcol-sumo
+```
+
+After successful execution of the above command, Sumo will start receiving the data from your host machine.
+
+Click **Next**. This will install the app to your Sumo Logic Org. The app consists of Dashboards.
+
+Panels will start to fill automatically. It's important to note that each panel slowly fills with data matching the time range query and received since the panel was created. Results won't immediately be available, but within 20 minutes, you'll see full graphs and map
```

Comment:
```suggestion
Panels will start to fill automatically. It's important to note that each panel slowly fills with data matching the time range query and received since the panel was created. Results won't immediately be available, but within 20 minutes, you'll see full graphs and maps.
```